.gitlab-ci.yml

```diff
@@ -3860,38 +3861,38 @@ latest_release_7:
 #
 # Use these steps to revert the latest tags to a previous release
 # while maintaining content trust signatures
-# - remove the leading dot from the name
-# - set the RELEASE envvar
+# - Create a pipeline on master with the RELEASE_6 and/or RELEASE_7 env vars
 # - in the gitlab pipeline view, trigger the step (in the first column)
 #
-
-.latest_revert_to_previous_release_6:
+revert_dockerhub_latest_6:
   rules:
-    - when: manual
+    - <<: *if_master_branch
+      when: manual
       allow_failure: true
   <<: *docker_tag_job_definition
-  stage: source_test
+  stage: maintenance_jobs
   variables:
     <<: *docker_hub_variables
-    RELEASE: ""  # tag name of the non-jmx version, for example "6.9.0"
+    RELEASE_6: ""  # tag name of the non-jmx version, for example "6.9.0"
```

Comment:
Nit: Can this be something more explicit like `RELEASE_TO_REMOVE_6`?

Reply:
It's not the release to remove but the target release that will replace `latest` and `6/7`.
I can try to find a better name though

Reply:
True. What about `NEW_LATEST` maybe?

Reply:
I like it, updated